lab-traceyr/README.me

```diff
@@ -0,0 +1,9 @@
+#Instructions
+
+-Once you download the file, in your root directory of this file, open node server.js in your terminal
+-In another terminal window open telnet localhost 3000
+-In a third terminal window open telnet localhost 3000
+-Use these two telnet windows to chat between each other
+
```

Comment:
Instructions on how to leave the chat session (by typing 'END' in your case) would be nice too.